code/modules/food/recipes_microwave.dm

```diff
@@ -2775,3 +2775,25 @@
 		/obj/item/weapon/reagent_containers/food/snacks/grown/peanut,
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/peanutbutter
+
+/datum/recipe/escargot
+	reagents = list (SODIUMCHLORIDE = 1)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/meat/snail,
+		/obj/item/clothing/head/helmet/snail_helm,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/garlic,
```

Comment:
since these are already gated behind cargo, why not substitute butter for garlic in the recipe? that way you only have to interact with JUST cargo not both cargo AND traders

Reply:
Good point